elide-async/src/test/java/com/yahoo/elide/async/service/AsyncExecutorServiceTest.java

```diff
@@ -27,29 +31,38 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class AsyncExecutorServiceTest {
 
     private AsyncExecutorService service;
     private Elide elide;
     private AsyncQueryDAO asyncQueryDao;
+    private AsyncQuery queryObj1;
+    private AsyncQuery queryObj2;
+    private AsyncQuery queryObj3;
+    private User testUser;
+    private AsyncQueryUpdateThread asyncQueryUpdateThread;
 
     @BeforeAll
-    public void setupMocks() {
+    public void setupMockElide() {
         HashMapDataStore inMemoryStore = new HashMapDataStore(AsyncQuery.class.getPackage());
         Map<String, Class<? extends Check>> checkMappings = new HashMap<>();
 
         elide = new Elide(
                 new ElideSettingsBuilder(inMemoryStore)
                         .withEntityDictionary(new EntityDictionary(checkMappings))
                         .build());
-
         asyncQueryDao = mock(DefaultAsyncQueryDAO.class);
-
+        queryObj1 = mock(AsyncQuery.class);
```

Comment:
THere is no reason to make these mocks global - they can just be created in each test function.